frontend/app/src/index.tsx

```diff
@@ -28,6 +28,9 @@ import ThemedApp from "./ThemedApp"
 
 const engine = new Styletron({ prefix: "st-" })
 
+// Timestamp when React is loaded for GUEST_READY message
+window.ReactLoad = new Date()
```

Comment:
should we prefix this as well with `streamlit` to avoid clashes in the global namespace or is `ReactLoad` an official variable convention for this?